k8s/perftest/common/ccd/user-profile-api.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: ccd
   annotations:
     flux.weave.works/automated: "true"
-    flux.weave.works/tag.java: glob:pr-256-*
+    flux.weave.works/tag.java: glob:perftest-*
```

Comment:
did you check docker has `perftest-* ` tag? I think you need corresponding PR branch tag